WordPress/src/test/java/org/wordpress/android/ui/reader/ReaderPostDetailsUiStateBuilderTest.kt

```diff
@@ -37,6 +37,8 @@ class ReaderPostDetailsUiStateBuilderTest {
         this.feedId = 2L
```

Comment:
Suggestion (💡): I know this is not a change related to this PR, but I thought I should comment anyway, in case you want to apply these suggestions:
1. The `ReaderPostDetailsUiStateBuilderTest` doesn't match the main class name, which is the `ReaderPostDetailUiStateBuilder`. There is a slight diff of `Details` vs. `Detail`. This is important because without aligning those two the IDE would not be able to associate those two together.
2. The `..., then ...` is missing from the `given/when/then` test pattern. You might want to add it.
3. On the `given` part of the test pattern, instead of doing `title` and `null title`, I would recommend `with title` and `without title`. Some apply for the `feature image url, I would add the `with featured image url`.

Reply:
Applied all suggestions:
1. In e3c1b08
2. In cb1b7aa
3. In bc9276f